### ISSUE TITLE
Add users search by idp alias to Keycloak admin UI

### ISSUE THREAD

**Issue by s-solianyk-onventis** [keycloak/keycloak#25997]
### Description

In some cases, there may be many different identity providers in the realm. Then it is very useful to search for users by idp alias in the Keycloak admin UI.
Search in the Keycloak admin interface has been updated to look like this in version 22.0.5:
![image](https://github.com/keycloak/keycloak/assets/120454144/31412103-b98a-4e75-8682-1177d468390d)
The Keycloak BE endpoint has an option to search by idp alias, but it would be nice to have this search option in the Keycloak admin UI.
As an idea, one more element could be added to the search parameters: idp alias.

### Motivation

Improve the usability of Keycloak admin UI user search

### Details

The following endpoint is used by the Keycloak admin UI to find users: `admin/realms/onventis/ui-ext/brute-force-user`
This endpoint accepts the idpAlias parameter to search by idp:
![image](https://github.com/keycloak/keycloak/assets/120454144/33e673a6-60e3-471a-a4e9-e02e5885e454)

**Comment by bfyxzls** [comment]
I changed the code to equal query, because in a big data environment, this like query is very bad for performance

**Comment by ssilvert** [comment]
@bfyxzls I agree this would be a nice enhancement.  Would you be willing to submit a PR for it?
